src/Common/tests/System/Net/Http/LoopbackServer.AuthenticationHelpers.cs

```diff
@@ -248,11 +248,11 @@ internal static bool IsDigestAuthTokenValid(string clientResponse, string reques
             }
 
             if (string.IsNullOrEmpty(algorithm))
-                algorithm = "sha-256";
+                algorithm = "MD5";
```

Comment:
This updates the loopback server to use the correct default algorithm as specified by RFC 7616: `If this is not present, it is assumed to be "MD5".` The client-side code already used the correct default.